ownphotos/settings.py

```diff
@@ -228,11 +240,19 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.11/howto/static-files/
 
+# Allow to define data folder like /var/lib/librephotos
+
+BASE_DATA = os.environ.get('BASE_DATA', '')
```

Comment:
This should be BASE_DATA = os.environ.get('BASE_DATA', '/'), because otherwise it will not find the folder under docker